mkdocs/docs/HPC/torque_frontend_via_jobcli.md

```diff
@@ -0,0 +1,131 @@
+
+## jobcli
+
+Jobcli was developed to make it possible for the {{hpcinfra}} to use a Torque frontend and a Slurm backend. In addition to that, it adds some additional options for Torque commands.
+
+### help option
+
+Adding `--help` to a Torque command when using it on the {{hpcinfra}} will output a manual for that command, including all possible options for that command (including the original ones from Torque and the ones added by jobcli) and an explanation for each one.
+
+### dryrun option
+
+Adding `--dryrun` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. Using `--dryrun` will not actually submit the job.
+
+See [the examples](./#examples)
+
+### debug option
+
+Similarly to `--dryrun`, adding `--debug` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. However in contrast to `--dryrun`, using `--debug` will actually submit the job afterwards.
+
+See [the examples](./#examples)
+
+### Examples
+
+The following examples illustrate the working of the `--dryrun` and `--debug` options with and example jobscript.
+
+`example.sh`:
+
+```shell
+#/bin/bash
+#PBS -l nodes=1:ppn=8
+#PBS -l walltime=2:30:00
+
+module load SciPy-bundle/...
+
+python script.py > script.out.${PBS_JOBID}
+```
+
+#### Example of the dryrun option
+
+Running the following command:
+
+```shell
+$ qsub --dryrun example.sh -N example
+```
+
+will generate this output:
+
+```shell
+
+Command that would have been run:
+---------------------------------
+
+/usr/bin/sbatch
+
+Job script that would have been submitted:
+------------------------------------------
+
+#!/bin/bash
+#SBATCH --chdir="/user/gent/464/vsc46416"
+#SBATCH --error="/kyukon/home/gent/464/vsc46416/examples/%x.e%A"
+#SBATCH --export="NONE"
+#SBATCH --get-user-env="60L"
+#SBATCH --job-name="example"
+#SBATCH --mail-type="NONE"
+#SBATCH --nodes="1"
+#SBATCH --ntasks-per-node="8"
+#SBATCH --ntasks="8"
+#SBATCH --output="/kyukon/home/gent/464/vsc46416/examples/%x.o%A"
+#SBATCH --time="02:30:00"
+
+### (start of lines that were added automatically by jobcli)
+#
+# original submission command:
+# qsub --dryrun example.sh -N example
+#
+# directory where submission command was executed:
+# /kyukon/home/gent/464/vsc46416/examples
+#
+# original script header:
+# #PBS -l nodes=1:ppn=8
+# #PBS -l walltime=2:30:00
+#
+### (end of lines that were added automatically by jobcli)
+
+#/bin/bash
+
+module load SciPy-bundle/2023.11-gfbf-2023b
+
+python script.py > script.out.${PBS_JOBID}
+```
+This output consist of a few components. For our example the most important lines are the ones that start with `$SBATCH` since these contain the translation of the Torque commands to Slurm commands. For example the job-name is the one we specified with the `-N` option in the command.
```

Comment:
You should clearly mention here that the body of the job script is not touched at all: only the header is tweaked, based on the `#PBS` lines in the job script and the options passed to `qsub`.

If a job script uses PBS-isms like `$PBS_JOBID`, they are retained.
Slurm is configured such on the HPC-UGent infrastructure that common `$PBS_*` environment variables are defined in the job environment, next to the Slurm equivalents (this part is not actually done by jobcli currently, but via a Slurm plugin, see also https://github.com/hpcugent/jobcli/issues/19)